binancial/strategies/volume_confirmed_momentum_crossover.py

```
@@ -0,0 +1,53 @@
+def volume_confirmed_momentum_crossover(ticker,
```

Ruff (0.8.2)

Missing return type annotation for public function `volume_confirmed_momentum_crossover`

(ANN201)